addons/mail/static/src/emoji_picker/emoji_picker.js

```diff
@@ -187,6 +202,18 @@ export class EmojiPicker extends Component {
         });
     }
 
+    get itemsAmount() {
+        return this.recentEmojis.length + this.getEmojis().length;
```

Comment:
I feel like we don't have to call a heavy func like `get recentEmojis()`, and `getEmojis()` just to calculate an amount especially if it triggered every time calling Keydown. maybe better keep the amount in an field and update when needed？

Reply:
Whenever the search value changes, the amount changes.
Also recent category might change whenever an emoji is used.